Stylesheets/authentication.css

```diff
@@ -0,0 +1,57 @@
+.auth-main-container {
+    min-width: 33rem;
+    margin: auto;
+    position: absolute;
+    left: 43rem;
+    top: 16rem;
+    padding: 2rem;
+    border-radius: 4px;
```

Comment:
from what i understand you are trying to center the login form, giving these values might affect when you resize. I think transform properties can help you better to center you form

Reply:
Adding to what Aishwarya mentioned, You can also do margin 0 auto and remove the absolute positioning values. That would do centre the container as well.

Reply:
done have changed it to margin : auto